.buildkite/pipeline.yml

```diff
@@ -4,6 +4,9 @@ agents:
   useVault: true
   image: family/enterprise-search-ubuntu-2204-connectors-py
 
+env:
+  DOCKERFILE_FTEST_PATH: "Dockerfile.ftest.wolfi"
```

Comment:
any reason not to just replace the existing `Dockerfile.ftest` with the new Wolfi-based one at this point, assuming the functional tests all run and pass using the new Wolfi-based image there?